File: twoSum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        result = {}
        for i in range(len(nums)):
            if nums[i] in result: 
                return result[nums[i], i]
            result[target - nums[i]] = i

File: javascript solutions/twoSum.js
var twoSum = function(nums, target) {
    let numberIndex = new Map ();
      let result = [];
      
      for(let i = 0; i < nums.length; i++) { 
          let num = nums[i];
          let neededNum = target - num;
          
          if (numberIndex.has(neededNum)) { 
              result[0] = numberIndex.get(neededNum);
              result[1] = i;
              
              return result
          }
          numberIndex.set(num, i);s
      }
      
      
      return result;
      
  };